src/components/focusManager.js

```diff
@@ -40,7 +40,7 @@ define(['dom'], function (dom) {
 
         try {
             element.focus({
-                preventScroll: true
+                preventScroll: false
```

Comment:
Maybe this fixes some the the webOS problems? @oddstr13

Reply:
According to chat logs this didn't fix anything for WebOS.